blockchain/blockchain-wallet-v4-frontend
# feat(recovery): Store and load wallet credentials in metadata

## Description (optional)
Going forward, with the first login, the wallet will store the credentials for login, as in, guid, password and sharedKey in metadata. As such, using the mnemonic it will be possible to login to an existing wallet, even when 2FA has been lost. Doing it this way will keep all information related to nabu and the custodial part of the wallet, as the user is literally logging into the very same wallet. Previously it would create a new wallet instead with the old mnemonic. 

## Testing Steps (optional)
Happy to write a few tests if someone gives me a pointer on what should be tested ideally. 

For manual testing, use an old wallet, go through mnemonic recovery and discover that the GUID has changed. Upon logging in with this build, going through the same steps you should discover that the GUID has remained the same
